classes/class-bsf-custom-fonts-render.php

```diff
@@ -242,7 +252,7 @@ public function add_style() {
 					$this->render_font_css( $load_font_name );
 				}
 				?>
-				<style type="text/css">
+				<style type="text/css" class="cst_fonts" id="cst_font_data">
```

Comment:
You can remove class from here as its not used anywhere.

Reply:
Unused class is removed now, Thanks!
Comment Resolved.